beacon-chain/sync/subscriber.go

```diff
@@ -109,19 +134,26 @@ func (s *Service) registerSubscribers(epoch primitives.Epoch, digest [4]byte) {
 			s.syncContributionAndProofSubscriber,
 			digest,
 		)
+
 		if flags.Get().SubscribeToAllSubnets {
-			s.subscribeStaticWithSyncSubnets(
+			s.subscribeStaticWithSubnets(
+				"Sync committee",
```

Comment:
We already add in the `SyncCommitteeSubnetTopicFormat` topic, so do we need to add this again ?

Reply:
See other comment.